### ISSUE TITLE
Problems with DRM from Kindle for PC

### ISSUE THREAD

**Issue by qleaper** [apprenticeharper/DeDRM_tools#395]
Hello, I have been successfully DRMing books for years using caliber and Kindle for PC. Now all of a sudden I am having issues. I updated calibre and installed the latest plug-in. I also uninstalled my kindle for PC and re-installed it. The problem is that the books are not importing to caliber properly. They are being imported as EBOK -- so no book. While before the files in My Kindle content had one EBOK.azw file, now there are two: the EBOK.azw and a file ending in .md -- which I supposed is the metadata. The files downloaded via USB from amazon will not de-DRM either. Any idea what is going on?

**Comment by qleaper** [comment]
I was able to fix it, so here it is for anyone having the same problem. The EBOK is a KFX file. I followed the command prompt instructions in this site:
https://www.mobileread.com/forums/showthread.php?t=283371

Method 2 - Disable KFX support in Kindle for PC/Mac

As an alternative, a method is available to prevent versions 1.19, 1.20 and 1.21 from making use of KFX format.
Exit the Kindle program.
Open a command window (PC) or terminal window (Mac).
Cut and paste the appropriate command for your operating system to disable the renderer-test program.
PC:
Code:
ren %localappdata%\Amazon\Kindle\application\renderer-test.exe renderer-test.xxx
Mac:
Code:
chmod -x /Applications/Kindle.app/Contents/MacOS/renderer-test

Then I reopened Kindle for PC, imported the books again, and they were able to transfer to Calibre in the correct format to be de-DRMed.
